doc/user/content/sql/create-source/postgres.md

```diff
@@ -67,7 +40,7 @@ Before creating the source in Materialize, you must:
 
 Once you create a materialized source from the publication, the source will contain the raw data stream of replication updates. You can then break the stream out into views that represent the publication's original tables with [`CREATE VIEWS`](/sql/create-views/). You can treat these tables as you would any other source and create other views or materialized views from them.
 
-#### Postgres schemas
+### Postgres schemas
```

Comment:
I believe that these should still be considered logically one level below the "PostgreSQL source details"?